lib/material/tests/test_mparagraph_aligns.flow

```diff
@@ -0,0 +1,22 @@
+import material/material;
+import material/material_manager;
+import material/material_dialog;
+
+main() {
+	setLang("ar");
+    manager = makeMaterialManager([]);
+    text = "Science, my boy, is made up of mistakes, but they are mistakes which it is useful to make, because they lead little by little to the truth.";
```

Comment:
Could you check how "\n" symbol inside the text will be honored?

Reply:
Tested with \n, works fine. 
![image](https://user-images.githubusercontent.com/27855185/63333759-a0877300-c342-11e9-8f14-79f527eb6eff.png)